qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -102,6 +102,11 @@ def __init__(
         if seed is not None:
             algorithm_globals.random_seed = seed
 
+        if C > 0:
+            self.C = C
+        else:
+            raise ValueError("C has to be a positive number.")
```

Comment:
```suggestion
            raise ValueError(f"C has to be a positive number, found {C}.")
```
Normally we like to include the failing value in case that can be helpful for a user seeing the error. SO maybe something like the suggestion